hal/armv7m/imxrt/117x/config.h

```diff
@@ -17,19 +17,23 @@
 #ifndef _CONFIG_H_
 #define _CONFIG_H_
 
+#ifndef __ASSEMBLY__
+
 #include "imxrt.h"
 #include "peripherals.h"
 #include "../types.h"
 #include "../../mpu.h"
 #include "../../string.h"
 
+#endif
+
 
 #define PATH_KERNEL "phoenix-armv7m7-imxrt117x.elf"
 
-/* Addresses descriptions */
-#define ADDR_SYSPAGE 0x202c0000
+#define SIZE_PAGE    0x200
+#define SIZE_STACK   8 * SIZE_PAGE
+#define SIZE_SYSPAGE 8 * SIZE_PAGE
 
-#define SIZE_STACK 5 * 1024
-#define SIZE_PAGE  0x200
+#define ADDR_STACK _end + SIZE_SYSPAGE + SIZE_STACK
```

Comment:
As before, it's a good idea to use brackets

Reply:
I've forgotten about making change it these files as well. Fixed.